### ISSUE TITLE
Error in asNamespace(ns) : not a namespace

### ISSUE THREAD

**Issue by robiRagan** [eduardoRubioG/MinimumRcpp#1]
```
==> Rcpp::compileAttributes()

* Updated R/RcppExports.R

==> R CMD INSTALL --preclean --no-multiarch --with-keep.source MinimumRcpp-master

* installing to library ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library’
* installing *source* package ‘MinimumRcpp’ ...
** using staged installation
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
** libs
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -Wall -g -O2  -c findParetoSet.cpp -o findParetoSet.o
clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Library/Frameworks/R.framework/Versions/3.6/Resources/library/Rcpp/include" -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include  -fPIC  -Wall -g -O2  -c kintersection.cpp -o kintersection.o
clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/lib -o MinimumRcpp.so RcppExports.o findParetoSet.o kintersection.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Library/Frameworks/R.framework/Versions/3.6/Resources/library/00LOCK-MinimumRcpp-master/00new/MinimumRcpp/libs
** R
** byte-compile and prepare package for lazy loading
Error in asNamespace(ns) : not a namespace
Error: unable to load R code in package ‘MinimumRcpp’
Execution halted
ERROR: lazy loading failed for package ‘MinimumRcpp’
* removing ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/MinimumRcpp’
* restoring previous ‘/Library/Frameworks/R.framework/Versions/3.6/Resources/library/MinimumRcpp’

Exited with status 1.
```

**Comment by robiRagan** [comment]
Maybe it has something to do with this: https://stackoverflow.com/questions/37555870/roxygen2-error-in-loadnamespacename-there-is-no-package-called-testthat

**Comment by eduardoRubioG** [comment]
Good news & bad news: 

Good news is that it's a reproducible error -- I'm getting the same output when I clone the repo. Bad news is that this means it's not working when you clone the repo. I'm not sure what got lost in translation from my local files to GitHub such that it fails to properly load when installing a clean repo. I'm not as savyv with `.Rproj` or R packages here so I'm not entirely sure where this could be going wrong but I am working on it as we speak!

**Comment by eduardoRubioG** [comment]
Dirk to the rescue -- 

It was an issue with my directory structure. Before I pushed the directory up, in an attempt to tidy things up, I moved the script test file into the `.R` directory. I didn't know that in an R package this directory was completely sourced during compilation. 

[Here is a Stack Overflow post I made earlier in which Dirk was able to help out. ](https://stackoverflow.com/questions/61468265/issues-compiling-rpackage-error-in-asnamespacens-using-rcpp)

Anyway. Merging the `namespace-hotfix` branch into master as I think it looks like everthing is working smoothly again. 

Let me know if you have more issues.
